lint-tasks.yml

```
@@ -128,6 +153,611 @@ tasks:
     internal: true
     cmd: "{{.ROOT_DIR}}/tools/yscope-dev-utils/lint-configs/symlink-cpp-lint-configs.sh"
 
+  check-cpp-static-diff:
+    sources: *cpp_source_files
+    deps:
+      - "cpp-lint-configs"
+      - ":core-generate"
```

```suggestion
      - ":core-generate"
      - "cpp-lint-configs"
```
We still consider the `:` when sorting.